package.json

```diff
@@ -41,13 +41,14 @@
   },
   "peerDependencies": {
     "intl": "^1.2.5",
-    "react-intl": "^2.8.0"
+    "react-intl": "^2.8.0 || ^5.0.0"
   },
   "devDependencies": {
     "@cerner/eslint-config-terra": "^4.0.0",
     "eslint": "^6.1.0",
     "jest": "^24.7.1",
-    "memory-fs": "^0.4.1"
+    "memory-fs": "^0.4.1",
+    "react-intl": "^2.8.0 || ^5.0.0"
```

Comment:
Which version gets installed locally for development?

Reply:
good question. we should probably pick a version

Reply:
I believe it's pulling the latest version. No point in having the or there for the dev dependencies. Will update